src/Forms/ConfirmedPasswordField.php

```diff
@@ -562,9 +571,33 @@ public function saveInto(DataObjectInterface $record)
             return;
         }
 
-        if (!($this->canBeEmpty && !$this->value)) {
-            parent::saveInto($record);
+        // Create a random password if password is blank
+        if ($this->canBeEmpty && !$this->value) {
+            $this->value = $this->createStrongRandomPassword();
         }
+
+        parent::saveInto($record);
+    }
+
+    private function createStrongRandomPassword(): string
```

Comment:
Move this to `Member` or the member form or wherever allows us to do this exclusively for new `Member` records being created through the CMS.

Reply:
Updated